build

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+
+####################
+# Set up defaults
+###################
+
+: ${DISTRO:?"Need to set DISTRO"}
+: ${ARCH:?"Need to set ARCH"}
+
+# Defaults for jenkins builds and local builds
+export LOG_LEVEL="debug"
```

Comment:
You shouldn't default to debug, i've noticed in the past that it slows the build down by a significant factor